### ISSUE TITLE
Connection string in PostgresRepo, Chapter 6.7

### ISSUE THREAD

**Issue by ivanovmg** [pycabook/pycabook#59]
I could be wrong, but it seems that the connection string in ``PostgresRepo`` shown in the book is incorrect.

When I used the snippet provided in the book, the database connection failed.

``` python
class PostgresRepo:
    def __init__(self, connection_data):
        connection_string = "postgresql+psycopg2://{}:{}@{}/{}".format(
            connection_data["user"],
            connection_data["password"],
            connection_data["host"],
            connection_data["dbname"],
        )
        ...
```

So I changed it to this: changed keys to the same used in the configuration json and added port, which was missing in the original listing.

``` python
class PostgresRepo:
    def __init__(self, connection_data):
        connection_string = 'postgresql+psycopg2://{}:{}@{}:{}/{}'.format(
            connection_data['POSTGRES_USER'],
            connection_data['POSTGRES_PASSWORD'],
            connection_data['POSTGRES_HOSTNAME'],
            connection_data['POSTGRES_PORT'],
            connection_data['APPLICATION_DB'],
        )
        ...
```

Does it make sense?
Or is there something wrong from my side (I am a total noob in sqlalchemy)?

**Comment by ivanovmg** [comment]
From the code here https://github.com/pycabook/rentomatic/blob/ed2-c06-s05/rentomatic/repository/postgresrepo.py 
it is apparent that there is indeed a slightly incorrect listing in the book.
The connection string in the code repo matches with what I expected.

**Comment by lgiordani** [comment]
Thanks Maxim, I will fix it in the book

**Comment by lgiordani** [comment]
This has been solved in version 2.1.0. Thanks!
